cookbooks/fb_apt/recipes/default.rb

```diff
@@ -64,7 +64,18 @@
   mode '0644'
 end
 
+execute 'check keyring format' do
+  only_if 'file /etc/apt/trusted.gpg | grep -q keybox'
```

Comment:
You mean to be using `file` here, right?  You're looking for file type, not content?